api/logs/views.py

```diff
@@ -29,8 +32,24 @@ def get_log(self):
             )
         return log
```

Comment:
It looks like we call the new `check_permissions` method every time that we call `get_log`. That suggest to me that we should push the `check_permissions` call down here into `get_log` itself, to ensure that it's always called, including for any future log node classes.